Admin.js

```diff
@@ -11,7 +11,7 @@
 
```

Comment:
- eslint: Line 38 exceeds the maximum line length of 80.
- eslint: Comments should not begin with a lowercase character